_field-types/metadata-fields/meta.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: default
+title: Meta
+nav_order: 30
+has_children: false
```

Comment:
This piece of metadata isn't needed, since the parent is already specified and this page contains no grandparents.

```suggestion
```